### PR TITLE
use cff standard for citation information

### PR DESCRIPTION
**Pull request by pmeier**
GH picks up on our `CITATION` file in the root of the repository.

Screenshot from 2022-10-04 11-34-54

However, [the preferred way](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) is use a `CITATION.cff` file instead since GH supports the [citation file format (CFF) standard](https://github.com/citation-file-format/citation-file-format). With this PR, the prompt changes to

Screenshot from 2022-10-04 13-48-21

with the following auto-generated bibtex entry:

```bibtex
@inproceedings{Paszke_PyTorch_An_Imperative_2019,
author = {Paszke, Adam and Gross, Sam and Massa, Francisco and Lerer, Adam and Bradbury, James and Chanan, Gregory and Killeen, Trevor and Lin, Zeming and Gimelshein, Natalia and Antiga, Luca and Desmaison, Alban and Kopf, Andreas and Yang, Edward and DeVito, Zachary and Raison, Martin and Tejani, Alykhan and Chilamkurthy, Sasank and Steiner, Benoit and Fang, Lu and Bai, Junjie and Chintala, Soumith},
booktitle = {Advances in Neural Information Processing Systems 32},
pages = {8024--8035},
publisher = {Curran Associates, Inc.},
title = {{PyTorch: An Imperative Style, High-Performance Deep Learning Library}},
url = {http://papers.neurips.cc/paper/9015-pytorch-an-imperative-style-high-performance-deep-learning-library.pdf},
year = {2019}
}
```

Comparing with what we currently have the only significant difference is that the editors are no longer listed although the metadata is there. This is an issue with GH's automatic conversion and might be fixed in the future. Plus, the cite key was changed from `NEURIPS2019_9015` to `Paszke_PyTorch_An_Imperative_2019`, but this has no effect on the rendered result.

Do we also want to adopt the CFF standard?

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,73 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
```
Comment (pmeier) [inline CITATION.cff:2]: This is for human consumption and could for example also include instructions how to generate the bibtex entry.

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+title: PyTorch
+authors:
+  - family-names: PyTorch Team
```
Comment (pmeier) [inline CITATION.cff:5]: The `authors` key is required and thus I've used the same string as we do in our package metadata:

https://github.com/pytorch/pytorch/blob/a75edfa97c9d985a337fbea7b9c0f4061153aaf0/setup.py#L1185

**Review by dagitses — APPROVED** [review]

**Comment by dagitses** [comment]
Thanks, this is great and appreciate the comments to help understand the PR.

**Comment by pmeier** [comment]
Maybe `merge -g` doesn't work if no CI jobs are triggered.

**Comment by pmeier** [comment]
@janeyx99 Could you have look at this PR? It seems I can't merge it with the bot. My guess is that it waits for CI to complete, but the change does not trigger any CI besides the CLA check.

**Comment by janeyx99** [comment]
ah sorry i just saw this! i see you’ve resolved it tho

**Pull request closed without merging**